src/instructlab/model/chat.py

```diff
@@ -600,6 +603,7 @@ def chat_model(
     logs_dir,
     vi_mode,
     visible_overflow,
+    models: List[cfg._model_config],
```

Comment:
hmmmm, so it would be preferable to keep config class types out of the `model` pkg since its a "cli" only related structure. 

but if its not possible that's fine. Could we resolve the model before getting down into this file and just pass it in?

Reply:
@cdoern This is just so we can do type-checking. How can we structure it so we can reuse the types later throughout the library?